crates/oxc_semantic/src/builder.rs

```diff
@@ -461,8 +469,10 @@ impl<'a> Visit<'a> for SemanticBuilder<'a> {
         }
 
         if !flags.is_top() {
-            self.bind_class_expression();
+            self.bind_function_or_class_expression();
         }
+
+        self.add_current_node_id_to_current_scope();
```

Comment:
Does this change close https://github.com/oxc-project/backlog/issues/65?

Reply:
Yes, but still need to think https://github.com/oxc-project/backlog/issues/65#issuecomment-2225275090